change/@ni-nimble-components-8499a62b-64ea-41e0-877c-6bacca7ae0c8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@msmithNI: @jattasNI @rajsite @atmgrifter00 interested in your thoughts on this implementation direction so far (but note the concerns I listed in the PR description).

Reply:
The general direction looks reasonable to me.

For the `cell-tabbable-children-change` I'm a bit curious how often that event is firing. Is it firing once per row continuously as you scroll for example? That could potentially be excessive object creation / gc pressure on scroll.

> In Chrome, if a focused element inside a cell is removed from the DOM, the table loses focus entirely.

Do you have a screencap of how that manifests? Not exactly clear how to try out the UX for that situation and if it's a new change in behavior or existing.

Reply:
No objections to the overall direction from me. (I left a couple comments while wrapping my head around the PR because I couldn't stop myself; neither one is existential, just normal code review feedback)

Reply:
When `cell-tabbable-children-change` fires:
- With what's in this PR, there's some unnecessary firing as the table loads with initial data. If we omit firing when `!cellView.$fastController.isConnected`, we could prevent that though.
- During scrolling:
  - Anchor column: Yes, depending on what's in the cells. With the Storybook example, yes, since the recycled cells change between having valid links / no link / placeholder text, as you scroll (depending on the data).
  - Menu button: Yes with this current PR. But not if we skip firing when the FAST controller is not connected.

Reply:
> Do you have a screencap of how that manifests?

Not currently, I was just going by the behavior of the new autotest failing in Chrome without the code to refocus the table. If we updated an example to periodically update the table data with an anchor column periodically switching between a valid URL / no URL, that would probably show it.